src/core/tests/Test_CoreMain.py

```diff
@@ -1109,7 +1103,6 @@ def test_delete_temp_folder_contents_failure(self):
         # mock os.remove()
         self.backup_os_remove = os.remove
         os.remove = self.mock_os_remove
-
```

Comment:
nit: Similar to the other comment on line breaks, this one separates mock of os.remove from the rest of the code. 
Line breaks are a way of sectioning code when the functions are large, to maintain readability.